argocd/keycloak/Chart.yaml

```diff
@@ -1,4 +1,17 @@
 ---
 apiVersion: "v2"
-name: "apps"
+name: "keycloak"
 version: "0"
+description: |
+  NOTE: This chart is enabled only when deploying DevOps Stack to K3s.
```

Comment:
Note sure. I think we also use it for SKS and maybe others eventually. Pretty much any cloud that doesn't provide OIDC out of the box #